docker-compose-hub.yml

```diff
@@ -54,6 +54,22 @@ services:
                 depends_on:
                         - plane-api
                         - plane-worker
+                        
+        public-deploy:
+                container_name: plane_public_deploy
+                build:
```

Comment:
@henit-chobisa can you also add the image attribute here since this will be using the pre-built images from docker-hub.

Reply:
Done and Pushed! ✅